<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rotateBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scalingBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scaling</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flipLRBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Left Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warpColBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Warp Cols</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cropBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flipTBBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Top Bottom</string>
   </property>
  </widget>
  <widget class="QLabel" name="view_result">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imgshowBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
